upload/controller.py

```diff
@@ -49,3 +49,17 @@ def update_package_check_finish(package_id):
 
     return False
 
+
+def update_package_check_errors(package_id):
+    package = get_object_or_404(Package, pk=package_id)
+    
+    resolutions_categories = set([
+        ve.resolution.action for ve in package.validationerror_set.all()
+    ])
+
+    if choices.ER_ACTION_TO_FIX not in resolutions_categories:
+        package.status = choices.PS_READY_TO_BE_FINISHED
+    else:
+        package.status = choices.PS_PENDING_CORRECTION
```

Comment:
talvez um for + break até encontrar ER_ACTION_TO_FIX

Reply:
Com break precisaria criar uma outra var para verificar se houve detecção da ação para então, a depender da coisa, setar um estado ou outro.

Reply:
Mas não seria:

```python
	package.status = choices.PS_READY_TO_BE_FINISHED
	for ve in package.validationerror_set.all():
		if ve.resolution.action == choices.ER_ACTION_TO_FIX :
			package.status = choices.PS_PENDING_CORRECTION
			break
```